pkg/apis/kops/cluster.go

```diff
@@ -244,6 +244,14 @@ type ClusterSpec struct {
 
 	// Hooks for custom actions e.g. on first installation
 	Hooks []HookSpec `json:"hooks,omitempty"`
+
+	// Alternative locations for files and containers
+	AssetSpec *AssetSpec `json:"assetSpec,omitempty"`
```

Comment:
We don't want the word spec in the JSON here, I don't think.  Probably just `assets`